commands/pull.js

```diff
@@ -117,14 +125,17 @@ const maybeTunnel = function * (herokuDb) {
   return herokuDb
 }
 
-const run = co.wrap(function * (sourceIn, targetIn) {
+const run = co.wrap(function * (sourceIn, targetIn, exclusions) {
+  
   yield prepare(targetIn)
 
   const source = yield maybeTunnel(sourceIn)
   const target = yield maybeTunnel(targetIn)
 
+  const exclude = exclusions.map(function(e) { return '--exclude-table-data=' + e }).join(' ')
```

Comment:
Why `--exclude-table-data` and not just `--exclude-table`? I'm not questioning the decision; I'm just curious about your use case and reasoning.

Reply:
Good question! My need for this feature is essentially for pulling apps databases with huge tables for local examination ; this tables contain some kind of 'advanced' logs with various criteria, so they are not crucial to the relation graph and cohesion of the database. Because pg:pull creates a new database, I would still like the table to be _created and structured_ for the app to be able to work in case I would like to perform some kind of debug. I do agree this is different from --exclude-table and options maybe should not be exclusive. In my case, I always use the heroku cli for pulling dbs it is always in the context of an application so there's always that kind of 'applicative' characterization to the operation.

Reply:
Okay, that makes sense. We can still add the other one later if people need it. I think it's okay to expose the underlying features piecemeal in this situation.

Reply:
Agreed :)